app/src/main/java/nerd/tuxmobil/fahrplan/congress/preferences/SharedPreferencesRepository.kt

```diff
@@ -5,55 +5,95 @@ import android.content.SharedPreferences
 import android.preference.PreferenceManager
 import androidx.core.content.edit
 import nerd.tuxmobil.fahrplan.congress.R
-import nerd.tuxmobil.fahrplan.congress.contract.BundleKeys
 
 class SharedPreferencesRepository(val context: Context) {
```

Comment:
I'd name this something like `SettingsRepository`. The fact that `SharedPreferences` are used is an implementation detail that's not important when using the repository.

Reply:
In the `AppRepository` layer I prefer to have this level of implementation detail.